### DISCUSSION TITLE
403 Forbidden after typing for a period of time in Firefox

### DISCUSSION THREAD

**Discussion by Ruin0x11** [standardnotes/forum#2785] [General]
After typing for about ten minutes in Firefox, I start to get performance issues and sync stops working, so I refresh the page. I then get a blank page with "403 Forbidden" in all my browsers and the desktop app. I can't access my notes or the settings page to backup my work during that time. After an arbitrary amount of time (about ten minutes) I regain access.

![2022-02-25 13_19_11-403 Forbidden - Chromium](https://user-images.githubusercontent.com/6700637/155804426-eae39f07-00dc-4144-acc8-ac2a4220288a.png)

**Comment by atmoio** [comment]
Hmm, sounds like you hit an infinite loop somehow and got timed out by the server. Which editor were you using when this happened? Just the plain editor or one of the specialty editors?

**Comment by Ruin0x11** [comment]
It was the plain editor, in the web and desktop versions.
